Document the repository move (#2283)
nrfconnect/sdk-nrf: doc/nrf/gs_installing.rst

   Run ``west update`` every time you change or modify the current working branch (for example, when you pull, rebase, or check out a different branch).
   This will bring the project repositories to the matching revision defined by the manifest file.

.. _repo_move:

> Pointing the repositories to the right remotes after they were moved

Reviewer: According to Joel, we're not supposed to document this in the installation instructions, but it's only supposed to be added to the release notes. Please discuss with him.

Author: @jdstapes

********************************************************************

> Short before |NCS| version 1.3.0 was released, the full set of repositories were moved from the ``NordicPlayground`` GiHub organization to their current home, the ``nrfconnect`` organization. They were also renamed, replacing the ``fw-nrfconnect-`` prefix with ``sdk-``.

Reviewer 2: ```suggestion
Shortly before |NCS| version 1.3.0 was released, the Git repositories were moved from the ``NordicPlayground`` GitHub organization to their current home, the ``nrfconnect`` organization. They were also renamed, replacing the ``fw-nrfconnect-`` prefix with ``sdk-``.
```

(The "set" removal is because otherwise it's the subject, which means the verbs would have to be singular IMO.)

> The full list of repositories with their original URL and the one they were moved to can be found in the table below:

Reviewer 2: ```suggestion
The full list of repositories with their old and new URLs can be found in the table below:
```

Just a suggestion, for brevity.

.. list-table::
   :header-rows: 1

   * - Old URL
     - New URL

   * - https://github.com/NordicPlayground/fw-nrfconnect-nrf
     - https://github.com/nrfconnect/sdk-nrf

   * - https://github.com/NordicPlayground/fw-nrfconnect-zephyr
     - https://github.com/nrfconnect/sdk-zephyr

   * - https://github.com/NordicPlayground/fw-nrfconnect-mcuboot
     - https://github.com/nrfconnect/sdk-mcuboot

   * - https://github.com/NordicPlayground/fw-nrfconnect-mcumgr
     - https://github.com/nrfconnect/sdk-mcumgr

   * - https://github.com/NordicPlayground/fw-nrfconnect-hal_nordic
     - https://github.com/nrfconnect/sdk-hal_nordic

   * - https://github.com/NordicPlayground/nrfxlib
     - https://github.com/nrfconnect/sdk-nrfxlib

If you cloned the repositories before the move took place you do not need to do anything, since any local repositories and even your own forks of the |NCS| repository set will automatically be redirected to the new ones. In the case however that you prefer them to point directly to their new locations, you can do that as well.

> You should start by renaming any personal forks that you may have of the |NCS| repositories to their new names. This is achieved by visiting your personal fork on a browser and editing the name there.  For example, to rename the old ``fw-nrfconnect-nrf`` repository, you would visit ``https://github.com/<username>/fw-nrfconnect-nrf``, click the "Settings" button and then under "Options", change the naem in the "Repository name" text field to ``sdk-nrf``.

Reviewer 2: ```suggestion
You should start by renaming any personal forks that you may have of the |NCS| repositories to their new names. This is achieved by visiting your personal fork on a browser and editing the name there.  For example, to rename the old ``fw-nrfconnect-nrf`` repository, you would visit ``https://github.com/<username>/fw-nrfconnect-nrf``, click the "Settings" button and then under "Options", change the name in the "Repository name" text field to ``sdk-nrf``.
```

     - https://github.com/nrfconnect/sdk-mcuboot

   * - https://github.com/NordicPlayground/fw-nrfconnect-mcumgr
     - https://github.com/nrfconnect/sdk-mcumgr

   * - https://github.com/NordicPlayground/fw-nrfconnect-hal_nordic
     - https://github.com/nrfconnect/sdk-hal_nordic

   * - https://github.com/NordicPlayground/nrfxlib
     - https://github.com/nrfconnect/sdk-nrfxlib

You should start by renaming any personal forks that you may have of the |NCS| repositories to their new names. This is achieved by visiting your personal fork on a browser and editing the name there.  For example, to rename the old ``fw-nrfconnect-nrf`` repository, you would visit ``https://github.com/<username>/fw-nrfconnect-nrf``, click the "Settings" button and then under "Options", change the name in the "Repository name" text field to ``sdk-nrf``.

Next, to rename the actual remotes, you can go into your local copy of each of the repositories listed in the table above and use the following command::

    git remote set-url <remote name> <new url>

For example to point your existing ``fw-nrfconnect-nrf`` clone to its new URL, you would do::

    cd ncs/zephyr

>     git remote set-url origin https://github.com/nrfconnect/sdk-nrf

Author: This is explained at length in the Development Model, so I will add a link to that section closer to this relevant part.
